meta-cli/src/cli/list.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+use super::serialize::orchestrate_serialization_workflow;
+use super::{Action, ConfigArgs};
+use crate::config::NodeConfig;
+use crate::deploy::actors::task_manager::TaskSource;
+use crate::interlude::*;
+use anyhow::Result;
+use async_trait::async_trait;
+use clap::Parser;
+use common::graphql::Query;
+use common::typegraph::Typegraph;
+use reqwest::Url;
+use serde::Deserialize;
+
+#[derive(Parser, Debug)]
+pub struct List {
+    #[clap(long)]
+    max_parallel_loads: Option<usize>,
+}
+
+#[async_trait]
+impl Action for List {
+    #[tracing::instrument]
+    async fn run(&self, args: ConfigArgs) -> Result<()> {
+        let dir = args.dir()?;
+        let task_source = TaskSource::Discovery(dir.clone().into());
+        let typegraphs = orchestrate_serialization_workflow(
+            args,
+            dir.clone(),
+            None,
+            self.max_parallel_loads,
+            task_source,
+        )
+        .await?;
+        self.display_typegraphs(dir, typegraphs).await
+    }
+}
+
+#[derive(Debug, Clone, Deserialize)]
+struct TypegraphInfo {
+    name: String,
+    url: String,
+}
+
+impl List {
+    async fn fetch_deployed_typegraphs(&self, dir: PathBuf) -> Result<Vec<TypegraphInfo>> {
+        let query = r#"
+            query {
+                typegraphs {
+                    name
+                    url
+                }
+            }
+        "#;
+        let mut node_config = NodeConfig::default();
+        node_config.url = Url::parse("http://localhost:7891")?;
```

Comment:
Can you add the same `--target` options like we have in `meta serialize`?

Reply:
yes !